### PR TITLE
Add lite-youtube-embed detection

### PR DESCRIPTION
**Pull request by kevinfarrugia**
GitHub page: https://github.com/paulirish/lite-youtube-embed (4.2k stars)
Demo: https://paulirish.github.io/lite-youtube-embed/

**node src/drivers/npm/cli.js https://paulirish.github.io/lite-youtube-embed/**
```
{
  "urls": {
    "https://paulirish.github.io/lite-youtube-embed/": { "status": 200 }
  },
  "technologies": [
    // ...
    {
      "slug": "lite-youtube-embed",
      "name": "lite-youtube-embed",
      "confidence": 100,
      "version": null,
      "icon": "YouTube.png",
      "website": "https://github.com/paulirish/lite-youtube-embed",
      "cpe": null,
      "categories": [
        {
          "id": 59,
          "slug": "javascript-libraries",
          "name": "JavaScript libraries"
        }
      ]
    }
    // ...
  ]
}

```

Sample pages (selected at random): 
- https://www.techbang.com/
- https://www.musicnotes.com/
- https://www.cgtrader.com/
- https://connectify.me/
- https://www.driversupport.com/
- https://www.canalturf.com/
- https://saucelabs.com/

Note: The technology does not have its own icon so I reused the YouTube icon.

### CODE REVIEW

File: src/technologies/l.json [hunk 1]
```diff
@@ -1563,5 +1563,14 @@
     },
     "oss": true,
     "website": "https://lit.dev"
+  },
+  "lite-youtube-embed": {
```
Comment (radum) [inline src/technologies/l.json:1567]: @slightlyoff made a lite vimeo also. Might be good to add it here also.

https://github.com/slightlyoff/lite-vimeo

Comment (kevinfarrugia) [reply src/technologies/l.json:1567]: True, but I do not think it meets the requirements as per https://github.com/wappalyzer/wappalyzer/blob/master/CONTRIBUTING.md#adding-a-new-technology.

If the maintainers want to have it added, I can create a new PR.

File: src/technologies/l.json [hunk 2]
```diff
@@ -1563,5 +1563,14 @@
...
+    "cats": [
+      59
+    ],
+    "description": "A faster youtube embed.",
```
Comment (nurbek91) [inline src/technologies/l.json:1571]: `"description": "The lite-youtube-embed  technique renders the YouTube video inside the IFRAME tag only when the play button in clicked thus improving the core web vitals score of your website.",`

File: src/technologies/l.json [hunk 3]
```diff
@@ -1563,5 +1563,14 @@
...
+    "html": "<(?:lite-youtube)",
```
Comment (nurbek91) [inline src/technologies/l.json:1572]: The html attribute is resource intensive so should be replaced with **js** and/or **scriptSrc** where possible: 
```
"js": {
      "LiteYTEmbed": ""
    },
    "scriptSrc": "/lite-yt-embed\\.js",

```
But if you want to check for a special tag anyway `"html": "<(?:lite-youtube)"` you can rewrite it like this: 
`"dom": "lite-youtube",`
But in our case  js and scriptSrc I think will be sufficient.

File: src/technologies/l.json [hunk 4]
```diff
@@ -1563,5 +1563,14 @@
...
+    "icon": "YouTube.png",
```
Comment (nurbek91) [inline src/technologies/l.json:1573]: The attribute of the icon can be removed, it's not the icon of this library.
or rewrite:  
"icon": "default.svg",

File: src/technologies/l.json [hunk 5]
```diff
@@ -1563,5 +1563,14 @@
...
+    "website": "https://github.com/paulirish/lite-youtube-embed"
```
Comment (nurbek91) [inline src/technologies/l.json:1574]: We can also add two additional useful attributes:
```
"oss": true, 
"implies": "YouTube",
```

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR!!!
Minor updates needed to perfect your request.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**